awx/ui_next/src/screens/Inventory/shared/data.hostFacts.json

```diff
@@ -0,0 +1,1249 @@
+{
```

Comment:
Seems like you have a file of essentially the same name in `/hosts/` could we remove 1 of them?

Reply:
agreed this is weird and feels like unnecessary duplication, but we don't want to share across screens.